api/src/telemetry/lib/init-telemetry.ts

```diff
@@ -0,0 +1,37 @@
+import { getCache } from '../../cache.js';
+import { useEnv } from '../../env.js';
+import { scheduleSynchronizedJob } from '../../utils/schedule.js';
+import { track } from './track.js';
+
+/**
+ * Exported to be able to test the anonymous callback function
+ */
+export const jobCallback = () => {
+	track();
+};
```

Comment:
Is there another way to test it without the need for this helper function in actual source code?

Reply:
Not really! I actually don't mind it as much. It makes the function down below way more legible 🤔 I'm secretly actually becoming a fan of having each piece so modular